src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -1129,6 +1129,9 @@
         az vm extension set -n VMAccessForLinux --publisher Microsoft.OSTCExtensions --version 1.4 \\
             --vm-name MyVm --resource-group MyResourceGroup \\
             --protected-settings '{"username":"user1", "ssh_key":"ssh_rsa ..."}'
+  - name: Add a customScript extension to a VM using --ids.
+    text: |
+        az vm extension set -n customScript --publisher Microsoft.Azure.Extensions --ids {vm_id}
```

Comment:
Do you have an example for using extension name?

Reply:
So far, no...